### ISSUE TITLE
Does not work on Windows 10

### ISSUE THREAD

**Issue by edalzell** [statamic/cli#11]
I don't expect this to be resolved (must be a small number of users), but thought it worthwhile to enter this so others may find it.

The CLI does not work on windows, with the following errors:

```
PHP Notice:  Undefined index: HOME in C:\Users\erin.dalzell\AppData\Roaming\Composer\vendor\statamic\cli\statamic on line 14

Notice: Undefined index: HOME in C:\Users\erin.dalzell\AppData\Roaming\Composer\vendor\statamic\cli\statamic on line 14
Statamic CLI Tool 1.0.3
```

and

```
In Process.php line 976:

  TTY mode is not supported on Windows platform.


update
```

**Comment by alwaysblank** [comment]
Can you still replicate this issue? i.e. is there a particular command that fails? I was able to run `php please` in Windows 10 w/ a fresh Statamic install without problems.

**Comment by edalzell** [comment]
`php please` is not same as `statamic update`. This issue is specifically about the statamic cli tool, NOT the `please` command.

**Comment by alwaysblank** [comment]
Sorry, I was tired and confused. Please take a look at that PR and let me know if it resolves the issues for you.

**Comment by jasonvarga** [comment]
Should be fixed in 1.0.5
